.circleci/config.yml

```diff
@@ -202,83 +236,358 @@ jobs:
+  check_package_rpm:
+    machine:
+      image: ubuntu-2204:current
+      resource_class: arm.medium
+    parameters:
+      arch:
+        type: string
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - add_ssh_keys:
+          fingerprints:
+            - 3a:d1:7a:b7:57:d7:85:0b:76:79:85:51:38:f3:e4:67
+      - checkout
+      - run: |
+          AWS_ACCESS_KEY_ID=$TEST_AWS_ACCESS_KEY_ID \
+          AWS_SECRET_ACCESS_KEY=$TEST_AWS_SECRET_ACCESS_KEY \
+          .circleci/scripts/package-validation/redhat << parameters.arch >> /tmp/workspace/artifacts/influxdb3*.<< parameters.arch >>.rpm
+  sign-packages:
+    circleci_ip_ranges: true
+    docker:
+      - image: quay.io/influxdb/rsign:latest
+        auth:
+          username: $QUAY_RSIGN_USERNAME
+          password: $QUAY_RSIGN_PASSWORD
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - fc:7b:6e:a6:38:7c:63:5a:13:be:cb:bb:fa:33:b3:3c
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: |
+          # We need this base so that we can filter it out of our checksums
+          # output and if that changes at all we only need to update it here
+          WORK_DIR="/tmp/workspace/artifacts/"
+          for target in ${WORK_DIR}*
+          do
+            case "${target}"
+            in
+              # rsign is shipped on Alpine Linux which uses "busybox ash" instead
+              # of bash. ash is somewhat more posix compliant and is missing some
+              # extensions and niceties from bash.
+              *.deb|*.rpm|*.tar.gz|*.zip)
+                rsign "${target}"
+              ;;
+            esac
 
-  # Compile cargo "release" profile binaries for influxdb3 & data generator
-  #
-  # Uses the latest ci_image (influxdb/rust) to build a release binary and
-  # copies it to a minimal container image based upon `rust:slim-buster`. This
-  # minimal image is then pushed to `quay.io/influxdb/influxdb3:${BRANCH}`.
-  build_release:
+            if [ -f "${target}" ]
+            then
+              # Since all artifacts are present, sign them here. This saves Circle
+              # credits over spinning up another instance just to separate out the
+              # checksum job.
+              sha256sum "${target}" | sed "s#$WORK_DIR##" >> "/tmp/workspace/artifacts/influxdb3-edge.${CIRCLE_TAG}.digests"
+
+              # write individual checksums
+              md5sum    "${target}" | sed "s#$WORK_DIR##" >> "${target}.md5"
+              sha256sum "${target}" | sed "s#$WORK_DIR##" >> "${target}.sha256"
+            fi
+          done
+      - persist_to_workspace:
+          root: /tmp/workspace
+          paths:
+            - artifacts
+      - store_artifacts:
+          path: /tmp/workspace/artifacts
+  publish-packages:
+    docker:
+      - image: cimg/python:3.6
+    parameters:
+      # "destination" should be one of:
+      #   - releases
+      #   - nightlies
+      #   - snapshots
```

Comment:
The `snapshots` bucket needs additional setup I think. From https://app.circleci.com/pipelines/github/influxdata/influxdb/38817/workflows/05b5eb9c-ef95-4d29-9465-c7dcd7a8a91a/jobs/362952:
```
upload failed: ../../../tmp/workspace/artifacts/influxdb3_3.0.0+snapshot-77fe062b-1_amd64.deb.sha256 to s3://dl.influxdata.com/influxdb/snapshots/influxdb3_3.0.0+snapshot-77fe062b-1_amd64.deb.sha256 An error occurred (AccessDenied) when calling the PutObject operation: Access Denied
```

Reply:
@bnpfeife could you help with that?

Reply:
Hey, I've created an issue with SRE: https://github.com/influxdata/sre/issues/3145

Reply:
All right, uploads into `dl.influxdb.com/influxdb/snapshots` are working now!

Reply:
@bnpfeife - thanks for setting this up! I see that there are a lot of influxdb3 packages in `influxdb/releases` still. Would you be able to move those over to `influxdb/snapshots`?

Reply:
All right, these have been moved!